### DISCUSSION TITLE
Game won't start

### DISCUSSION THREAD

**Discussion by Gonzalo-Bruna** [ppy/osu#19054] [Q&A]
for some reason the game isn't opening. I tried downloading the .exe in the lastest version and re-installing (without previously uninstalling) but it didn't help, it just created another icon in the desktop and it won't start.

My operating system is Windows 11 Pro Insider Preview version 22H2

Logs just in case, but anyway they are from the last time I opened lazer (except for the runtime log).

[performance-input.log](https://github.com/ppy/osu/files/9075582/performance-input.log)
[performance-update.log](https://github.com/ppy/osu/files/9075583/performance-update.log)
[runtime.log](https://github.com/ppy/osu/files/9075584/runtime.log)
[updater.log](https://github.com/ppy/osu/files/9075585/updater.log)
[network.log](https://github.com/ppy/osu/files/9075578/network.log)
[performance.log](https://github.com/ppy/osu/files/9075579/performance.log)
[performance-audio.log](https://github.com/ppy/osu/files/9075580/performance-audio.log)
[performance-draw.log](https://github.com/ppy/osu/files/9075581/performance-draw.log)
[database.log](https://github.com/ppy/osu/files/9075575/database.log)
[legacy-ipc.log](https://github.com/ppy/osu/files/9075577/legacy-ipc.log)

**Comment by Gonzalo-Bruna** [comment]
I found the problem. The only thing that changed in my setup lately was the addition of a new gamepad, one similar to the playstation 2 controller, and since it was connected to my pc, the game won't start. I am unsure why this new controller would block the initialization of the game but it does. I disconnected it and the game worked again, I closed the game and connected the controller again and it won't open again, so definitely that was the problem.

**Comment by Gonzalo-Bruna** [comment]
I tried connecting my new controller to the game when it was open and it crashed inmediately. Logs don't show much as far as I can see ( at least runtime), but here there are just in case.


[performance.log](https://github.com/ppy/osu/files/9075692/performance.log)
[performance-audio.log](https://github.com/ppy/osu/files/9075693/performance-audio.log)
[performance-draw.log](https://github.com/ppy/osu/files/9075694/performance-draw.log)
[performance-input.log](https://github.com/ppy/osu/files/9075695/performance-input.log)
[performance-update.log](https://github.com/ppy/osu/files/9075696/performance-update.log)
[updater.log](https://github.com/ppy/osu/files/9075698/updater.log)
[runtime.log](https://github.com/ppy/osu/files/9075697/runtime.log)
[network.log](https://github.com/ppy/osu/files/9075691/network.log)
[database.log](https://github.com/ppy/osu/files/9075689/database.log)
[legacy-ipc.log](https://github.com/ppy/osu/files/9075690/legacy-ipc.log)

**Comment by bdach** [comment]
Please attach Windows Event Viewer info by performing the following steps:
1. Type `Event viewer` in the Windows search bar (shortcut: <kbd>Windows</kbd>+<kbd>S</kbd>) and press <kbd>Enter</kbd>, or open the *Run...* button (<kbd>Windows</kbd>+<kbd>R</kbd>), type in `eventvwr` and press <kbd>Enter</kbd>.
2. The *Event viewer* window will appear. Expand *Windows Logs* on the left and choose *Application*. It might take a while to load.
3. In the *Actions* pane on the right click *Save All Events As...*, then choose a location to save to.
4. On the *Display Information* popup, select the second option, then the checkbox to show all available languages, then *English (UK)* (US would also do fine).
5. Append the resulting `.evtx` file to this issue.
